benchmarks-next/blake3-scalar/rust-benchmark/src/main.rs

```diff
@@ -11,13 +11,13 @@ extern "C" {
 
 fn main() {
     let buffer = [0 as u8; 64 << 10];
-    println!("[blake3] hashing a 64KB, zero-filled buffer");
+    eprintln!("[blake3] hashing a 64KB, zero-filled buffer");
```

Comment:
Where did the `benchmark.wasm` and `benchmark.wat` for this file go? Looks like they got deleted?

Reply:
Hmm... let me double check what happened here...

Reply:
They are still there, AFAICT?

Reply:
Ok, I guess I misread the diff... must have been just a bunch of lines deleted.